src/vaadin-scroll-container.html

```diff
@@ -61,6 +64,19 @@
           return 'vaadin-scroll-container';
         }
 
+        static get properties() {
+          return {
+            /**
+            * This attribute indicates the scroll direction. Supported values are `vertical`, `horizontal`, `none` or `both`. Defaults to `both`.
+            */
+            scrollDirection: {
+              type: String,
+              value: 'both',
```

Comment:
Avoid truthy default values for properties. "both" doesn't have any logic to it ("foobar" works the same) so maybe only list the supported values that affect something. Leaving the property `undefined` might be a better default.

Reply:
I think that if we don't want an explicit `both` value then we shouldn't have the property at all. The behavior is already documented in styling attributes section.

Reply:
Hmm...I'm not sure about not having the property. The attribute in this case does indeed all the heavy lifting. But we've typically preferred having properties for something a user might want to change.

For example, you should be able to say `layout.scrollDirection = "vertical"` instead of having to do a DOM manipulation with `layout.setAttribute('scroll-direction', 'vertical')`. Example: https://github.com/vaadin/vaadin-tabs/blob/next/src/vaadin-tabs-mixin.ts#L73

Same thing as with `component.disabled = true` (instead of `component.setAttribute('disabled', '')`) or the native `component.hidden = true`

Reply:
Added property back and adjusted doc